codec-http/src/test/java/io/netty/handler/codec/http/multipart/HttpPostRequestEncoderTest.java

```diff
@@ -443,4 +447,25 @@ private static void checkNextChunkSize(HttpPostRequestEncoder encoder, int sizeW
                 + readable, expectedSize);
         httpContent.release();
     }
+
+    @Test
+    public void testEncodeChunkedContent() throws Exception {
+        EmbeddedChannel ch = new EmbeddedChannel(new ChunkedWriteHandler());
```

Comment:
why is the `ChunkedWriteHandler` needed ?

Reply:
That was the closest thing I could find to the logic in Finagle. I just changed it though.